### PR TITLE
Update Presigned URL example to ditch jQuery (broken) in favor of ES6 (working)

### PR DESCRIPTION
**Pull request by Lewiscowles1986**
The jQuery example told me `$` was undefined

I switched this around so I could run locally against a golang backend to verify Minio. This is it being switched back, using modern JS ES6 with newer browser API's

## Browser compatibility

- [Promise API compatibility](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise#Browser_compatibility)
- [querySelector API compatibility](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector#Browser_compatibility)
- [Fetch API compatibility](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API#Browser_compatibility)
- [File API compatibility](https://developer.mozilla.org/en-US/docs/Web/API/File#Browser_compatibility)

### CODE REVIEW

File: docs/presigned-put-upload-via-browser.md [hunk 1]
```diff
@@ -56,51 +56,50 @@ The client-side web application's user interface contains a selector field that
...
-      var file = files[i]
-      // Retrieve a URL from our server.
-      retrieveNewURL(file, (file, url) => {
-        // Upload the file to the server.
-        uploadFile(file, url)
-      })
+        // Get selected files from the input element.
+        var files = document.querySelector("#selector").files
+        for (var i = 0; i < files.length; i++) {
+        var file = files[i]
```
Comment (kaankabalak) [inline docs/presigned-put-upload-via-browser.md]: Thanks for the contribution @Lewiscowles1986! The PR is working fine for me, however is it possible to indent the block of code inside the `for` loop if possible for better readability?

Comment (Lewiscowles1986) [reply docs/presigned-put-upload-via-browser.md]: Of course it is. I did notice yesterday but did not want to disrupt reading or review.

Comment (Lewiscowles1986) [reply docs/presigned-put-upload-via-browser.md]: I've also added semi-colons to the ends of some lines.

File: docs/presigned-put-upload-via-browser.md [hunk 2]
```diff
@@ -56,51 +56,50 @@ The client-side web application's user interface contains a selector field that
...
-        uploadFile(file, url)
-      })
+        // Get selected files from the input element.
+        var files = document.querySelector("#selector").files
+        for (var i = 0; i < files.length; i++) {
+            var file = files[i];
+            // Retrieve a URL from our server.
+            retrieveNewURL(file, (file, url) => {
+                // Upload the file to the server.
+                uploadFile(file, url)
```
Comment (kaankabalak) [inline docs/presigned-put-upload-via-browser.md]: There is just two very small semicolon issues left before approving, is it possible to add a semicolon at the end of this line?

File: docs/presigned-put-upload-via-browser.md [hunk 3]
```diff
@@ -56,51 +56,50 @@ The client-side web application's user interface contains a selector field that
...
-    var files = $("#selector")[0].files
-    for (var i = 0; i < files.length; i++) {
-      var file = files[i]
-      // Retrieve a URL from our server.
-      retrieveNewURL(file, (file, url) => {
-        // Upload the file to the server.
-        uploadFile(file, url)
-      })
+        // Get selected files from the input element.
+        var files = document.querySelector("#selector").files
```
Comment (kaankabalak) [inline docs/presigned-put-upload-via-browser.md]: Is it also possible to add a semicolon at the end of this line as well?

**Comment by Lewiscowles1986** [comment]
@kaankabalak it's been updated

**Pull request merged**